src/encoded/static/libs/ui/tooltip.js

```diff
@@ -14,107 +14,95 @@ import PropTypes from 'prop-types';
+    // Special case for menu tooltips on mobile to allow the pop-up to be displayed as position relative on devices smaller than 800px wide with touch screens
+    if (isMobileDevice && relativeTooltipFlag) {
         return (
-            <div className={wrapperCss}>
-                {this.state.tooltipDisplayed ?
-                    <div className={tooltipCss} role="tooltip" id={tooltipId} onMouseEnter={this.handleTooltipMouseEnter} onMouseLeave={this.handleTooltipMouseLeave}>
-                        <div className="tooltip-arrow" />
-                        <div className="tooltip-inner">{this.props.children}</div>
-                    </div>
-                : null}
+            <React.Fragment>
                 <button
-                    aria-describedby={this.state.tooltipDisplayed ? tooltipId : ''}
-                    onMouseEnter={this.handleTriggerMouseEnter}
-                    onMouseLeave={this.handleTriggerMouseLeave}
-                    onFocus={this.handleTriggerFocus}
-                    onBlur={this.handleTriggerBlur}
-                    className="tooltip-container__trigger"
+                    aria-describedby={showDefinition ? tooltipId : ''}
+                    onMouseEnter={e => setDefinitionVisibility(true, e)}
+                    onMouseLeave={e => setDefinitionVisibility(false, e)}
+                    onFocus={e => setDefinitionVisibility(true, e)}
+                    onBlur={e => setDefinitionVisibility(false, e)}
+                    className={`tooltip-container__trigger ${showDefinition ? 'show' : ''}`}
                 >
                     {trigger}
                 </button>
-            </div>
+                {showDefinition ?
+                    <div
+                        className={tooltipCss}
+                        role="tooltip"
+                        id={tooltipId}
+                    >
+                        <div className="tooltip-arrow" />
+                        <div className="tooltip-inner">{children}</div>
+                    </div>
+                : null}
+            </React.Fragment>
         );
     }
-}
+    return (
+        <div className={wrapperCss}>
+            <button
+                aria-describedby={showDefinition ? tooltipId : ''}
+                onMouseEnter={e => setDefinitionVisibility(true, e)}
+                onMouseLeave={e => setDefinitionVisibility(false, e)}
+                onFocus={e => setDefinitionVisibility(true, e)}
+                onBlur={e => setDefinitionVisibility(false, e)}
+                className={`tooltip-container__trigger ${showDefinition ? 'show' : ''}`}
+            >
+                {trigger}
+            </button>
+            {showDefinition ?
+                <div
+                    className={tooltipCss}
+                    role="tooltip"
+                    id={tooltipId}
+                >
+                    <div className="tooltip-arrow" />
```

Comment:
I was keeping it just because it was part of the original code. I've removed it, after verifying that it has no associated styles. We can always add it back. I assume the original thought was to have little carets above the tooltip pop-ups.